### DISCUSSION TITLE
A multi-paragraph text block

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1055] [Ideas]
Here's what seems off - that a content piece might require 25 individual text blocks to compose. Am I missing something?

Hitting the ENTER key inserts line breaks into the block. That won't work because the spacing applied will be inter-line instead of inter-paragraph spacing. So, that points to having an individual text block for each paragraph.

What I'd like to see is something like a multi-paragraph text block, where the ENTER key will insert a paragraph p tag instead of a line-break br tag and provide for having true paragraphs, with true inter-paragraph spacing.

**Comment by meetmandeep** [comment]
> Hitting the ENTER key inserts line breaks into the block. That won't work because the spacing applied will be inter-line instead of inter-paragraph spacing. So, that points to having an individual text block for each paragraph.

This can be easily addressed by theme styling. You can always add different padding/margin to `<br /> and <p>` tags. 

> What I'd like to see is something like a multi-paragraph text block, where the ENTER key will insert a paragraph p tag instead of a line-break br tag and provide for having true paragraphs, with true inter-paragraph spacing.

This will likely never happen because our editor will pick each p tag as a new text block.
